analysis/analysis/analysis.py

```diff
@@ -61,7 +61,9 @@
 
 log = logging.getLogger("analysis")
 metrics = setup_metrics(name="analysis")
-DB_URI = "clickhouse://api:api@localhost/default"
+DEFAULT_DB_URI = "clickhouse://api:api@localhost/default"
+CONF_FILE = Path("/etc/ooni/analysis.conf")
```

Comment:
Are these used anywhere? Maybe they can be removed?

Reply:
DEFAULT_DB here: https://github.com/ooni/backend/blob/c35ce2516b98b653cc994a6091bdd1eef65f7ac7/analysis/analysis/analysis.py#L86
 
 CONF_FILE is not, removing it